com.unity.render-pipelines.high-definition/Runtime/Material/Unlit/Unlit.hlsl

```diff
@@ -55,7 +55,11 @@ LightTransportData GetLightTransportData(SurfaceData surfaceData, BuiltinData bu
 {
     LightTransportData lightTransportData;
 
+#if (SHADERPASS == SHADERPASS_LIGHT_TRANSPORT)
+    lightTransportData.diffuseColor = surfaceData.color;
```

Comment:
I suppose it is to match builtin behavior?   We chose to not handle color in HDRP as color don't have any light value. HDRP rely on physical light value and thus, only Emissive color have it, we should avoid to have regular color in it. (Also I expect it may break various project)

Reply:
Right, in builtin, unlit shaders still render out the diffuse color. If HDRP intends to have different behavior for this view, then I'm ok to revert this.

Reply:
yes we should revert this one

Reply:
Done in fe87b58